tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -258,12 +258,10 @@ def parse_args():
         "processed.",
     )
 
-    return parser.parse_args()
+    return parser.parse_args() if args is None else parser.parse_args(args)
 
 
-def convert_onnx_models_to_ort():
-    args = parse_args()
-
+def convert_onnx_models_to_ort(args):
```

Comment:
it would be nice if this function took explicit parameters instead of the more opaque `args`.

if this approach is more convenient, it's ok. but then, I'd suggest keeping the argument validation (parse_args() call) inside this function instead of relying on the caller to call parse_args() first. I think we can just pass an optional list of command line argument strings with a default value of None from here all the way through to argparse.ArgumentParser.parse_args().

Reply:
My intention was to minimize the changes to this file. But I think I can update this function to accept parameters instead of argsparse args. Will make the change.